org.ekstep.preview-1.2/editor/style.css

```diff
@@ -25,23 +19,23 @@
     border: 0px;
 }
 
-.main {
```

Comment:
You have removed these classess fully. How did you confirm, it will not affect any other place when they are using this class?

Reply:
@vinukumar-vs 
All plugin are independent.
If someone else is reading same class from this plugin, we cannot handle those condition.

Reply:
If somebody using this plugin in their code, then their changes will break because of this right?

Reply:
This has to verify in